yarn.lock

```diff
@@ -960,6 +960,11 @@
   dependencies:
     "@types/node" ">= 8"
 
+"@okta/odyssey@^0.2.0":
```

Comment:
Hmm. This dependency callout shouldn't be here. Instead, the local package of `@okta/odyssey` should be symlinked.

Reply:
Idea on how to fix? This was the result of doing a `lerna create` and then a `lerna version 0.3.0` since the individual packages were out of sync with the master version.

Reply:
I'd assume another run of `yarn install` or `lerna bootstrap` should correctly remove the entry from this lock file. If not - try removing it by hand and run `yarn install` again to make sure it doesn't reappear.

Reply:
Found it, the docs package currently lists `"@okta/odyssey": "^0.2.0",` as a dev dependency, so any fresh builds of the lock file include that.

What's the current way to specify dev deps inside of a workspace?

Reply:
Ah, looks like `lerna bootstrap` should have detected this, but is currently bugged / behavior does not match the docs? https://github.com/lerna/lerna/issues/2352

Reply:
@rajaguruduraisamy-okta do you have a sense of how to fix this non-relative dependency?

Does it need to be redeclared in the docs package a different way? AFAIK the docs were moved over from `okta-ui` without adjusting their deps for the lerna project setup.